iroha/src/kura.rs

```diff
@@ -273,13 +326,16 @@ pub mod config {
         /// Path to the existing block store folder or path to create new folder.
         #[serde(default = "default_kura_block_store_path")]
         pub kura_block_store_path: String,
+        #[serde(default = "default_kura_blocks_per_storage_file")]
+        pub kura_blocks_per_storage_file: u64,
```

Comment:
Could you remove prefix kura from fields? Also add this after `serde` rename_all before struct:
```rust
#[config(env_prefix = "KURA_")]
```

Reply:
Is it related to kura_block_store_path as well?

Reply:
Yep, to all fields. It used for setting fields in configuration from shell environment variables.